pallets/loans/benchmarking/src/lib.rs

```diff
@@ -222,34 +261,39 @@ benchmarks! {
     }
 
     liquidate_borrow {
-        let borrower: T::AccountId = whitelisted_caller();
-        initial_set_up::<T>(borrower.clone());
-        let caller: T::AccountId = account("Sample", 100, SEED);
-        <T as LoansConfig>::Currency::deposit(DOT, &caller.clone(), INITIAL_AMOUNT).unwrap();
-        let repay_amount = 2000;
-        let borrowed_amount = 100_000_000;
-        let currency_id: <T as ORMLOracleConfig<Instance1>>::OracleKey = T::convert(DOT);
-        let price: <T as ORMLOracleConfig<Instance1>>::OracleValue = T::convert_price(PriceWithDecimal{ price: FixedU128::from(100_000), decimal: 12 });
-        assert_ok!(ORMLOracle::<T, _>::feed_values(SystemOrigin::Root.into(),
-            vec![(currency_id, price)]));
-        assert_ok!(Loans::<T>::mint(SystemOrigin::Signed(borrower.clone()).into(), DOT, INITIAL_AMOUNT));
-        assert_ok!(Loans::<T>::collateral_asset(SystemOrigin::Signed(borrower.clone()).into(), DOT, true));
-        assert_ok!(Loans::<T>::borrow(SystemOrigin::Signed(borrower.clone()).into(), DOT, borrowed_amount));
-        let total_borrows = pallet_loans::TotalBorrows::<T>::get(DOT);
+        let alice: T::AccountId = account("Sample", 100, SEED);
+        initial_set_up::<T>();
+        let bob: T::AccountId = account("Sample", 101, SEED);
+        transfer_initial_balance::<T>(alice.clone());
+        transfer_initial_balance::<T>(bob.clone());
+        assert_ok!(Loans::<T>::mint(SystemOrigin::Signed(bob.clone()).into(), KSM, 200_000_000_00));
+        assert_ok!(Loans::<T>::mint(SystemOrigin::Signed(alice.clone()).into(), DOT, 200_000_000_00));
+        assert_ok!(Loans::<T>::collateral_asset(SystemOrigin::Signed(alice.clone()).into(), DOT, true));
+        assert_ok!(Loans::<T>::borrow(SystemOrigin::Signed(alice.clone()).into(), KSM, 100_000_000_00));
+                let price = PriceWithDecimal {
+                    price: Price::saturating_from_integer(2),
+                    decimal: 8
+                };
+        assert_ok!(Prices::<T>::set_price(SystemOrigin::Root.into(), KSM, price));
     }: {
-         let _ = Loans::<T>::liquidate_borrow(SystemOrigin::Signed(caller.clone()).into(), borrower, DOT, repay_amount, DOT);
+         let _ = Loans::<T>::liquidate_borrow(SystemOrigin::Signed(bob.clone()).into(), alice.clone(), KSM, 50_000_000_00, DOT);
     }
     verify {
         assert_eq!(
-            pallet_loans::TotalBorrows::<T>::get(DOT),
-            total_borrows - repay_amount,
+            <T as LoansConfig>::Currency::free_balance(DOT, &alice),
+            800_000_000_00,
+        );
+        assert_eq!(
```

Comment:
liquidate_borrow has no effects on the free_balance of both Alive and Bob. I think it should check the borrow balance of Alice and the voucher_balance of Bob.

Reply:
Check Alice's borrow balance sample: https://github.com/parallel-finance/parallel/blob/master/pallets/loans/src/tests/liquidate_borrow.rs#L111

Reply:
The sample of checking Bob's voucher balance: https://github.com/parallel-finance/parallel/blob/master/pallets/loans/src/tests/liquidate_borrow.rs#L119

Reply:
Alice will reduce 50_000_000_00 borrow balance in this case and Bob will add 110_000_000_00 / Loans::exchange_rate(DOT) into voucher_balance.